port/linux/ipadapter.c

```diff
@@ -327,12 +327,14 @@ get_interface_addresses(ip_context_t *dev, unsigned char family, int port,
   while (!done) {
     ssize_t response_len = get_data_size(nl_sock);
     if (response_len < 0) {
+      OC_ERR("failed to get data size: %d", (int)-response_len);
```

Comment:
is it possible to use `strerror_r` ?

Reply:
yeah, it should be available in the linux port